src/command_modules/azure-cli-container/azure/cli/command_modules/container/_params.py

```diff
@@ -15,6 +15,7 @@
 
 IP_ADDRESS_TYPES = ['Public']
 OS_TYPES = ['Windows', 'Linux']
+RESTART_POLICIES = ['Always', 'OnFailure', 'Never']
```

Comment:
These are defined in the SDK.
Can't you use that?
https://docs.microsoft.com/en-us/python/api/azure.mgmt.containerinstance.models.containergrouprestartpolicy?view=azure-python